Datadog/Example/Debugging/DebugOTelTracingViewController.swift

```diff
@@ -0,0 +1,123 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import UIKit
+import DatadogCore
+import DatadogTrace
```

Comment:
nit/ In theory, we need `import DatadogTrace` only to configure the OTel tracer, but here, instrumentation should be possible with only `import OpenTelemetryApi`, right?